packages/heuristics/src/function-builder-detectors/netlify-lambda.js

```diff
@@ -0,0 +1,34 @@
+const { existsSync, readFileSync } = require('fs')
+
+const execa = require('execa')
+
+module.exports = function() {
```

Comment:
We are moving away from `netlify-lambda` as a bundling solution. 

Instead build plugins can handle custom webpack/babel/typescipt etc.

cc @ehmicky any ideas on what those function plugins might look like?

Reply:
I am not very familiar with how this is currently setup in the CLI. If I'm not mistaken, this code is taken from the CLI, correct? We could just re-use that code for the moment, and once this PR is merged, work on migrating away from `netlify-lambda` separately?